src/cpp/session/modules/NotebookData.R

```diff
@@ -143,6 +175,53 @@
 
 .rs.addFunction("readDataCapture", function(path)
 {
+  "%||%" <- function(x, y) {
```

Comment:
Is there a way we could factor the rmarkdown code so that we could call this code within the rmarkdown package? (since it's duplicated there). Note that it's fine for the IDE to call private methods of packages since we aren't subjected to CRAN checks.